schema.sql

```diff
@@ -0,0 +1,19 @@
+DROP DATABASE IF EXISTS photo_display;
+
+CREATE DATABASE photo_display;
+
+USE photo_display;
+
+CREATE TABLE photo_information (
+  id INT NOT NULL AUTO_INCREMENT,
+  photo_url VARCHAR(255) NOT NULL,
+  description VARCHAR(1000) NOT NULL,
+  is_verified BOOLEAN NOT NULL,
+  listing_id INT,
+  PRIMARY KEY (id)
```

Comment:
You can put 'PRIMARY KEY' in the line same line as the field to save one line of space, but this also works ^^